Xamarin.Forms.Platform.Android/VisualElementRenderer.cs

```diff
@@ -173,6 +173,10 @@ public override AView FocusSearch(AView focused, [GeneratedEnum] FocusSearchDire
 			if (tabIndexes == null)
 				return base.FocusSearch(focused, direction);
 
+			// use OS default--there's no need for us to keep going if there's one or fewer tab indexes!
+			if (tabIndexes.Count <= 1)
```

Comment:
Thanks for the quick response @samhouts !

Now I can see this change in place, will the `tabIndexes` indeed be empty when none are set?
Looking at `TabIndexExtensions`, `GetTabIndexesOnParentPage` only seems to care if `IsTabStop` is set, which is true by default for every control.

So will leaving `TabIndex` unset mean `GetTabIndexesOnParentPage` will return an empty list?

It would appear for this solution to work with us, we would have to leave in our "set IsTabStop to false" reflection hack which obviously we would prefer not to have.

Reply:
So, this is actually looking for TabIndex groups. If you haven't set them, the TabIndexes default to 0. In that case, this returns a single group of all elements with TabIndex == 0. If there's a single group, then there's no need to use this logic.

Reply:
Nice, does that mean that apps will work as they did before Xamarin 4.0 if they are not using tab stops @samhouts ?

Reply:
Ah.... makes sense. Could I get an out of band nuget for these changes so I could try them out? I think this entire branch would good to validate against our cases.

We are currently on 4.3, but I think I can test 4.4 without much problem

Reply:
@joshuangfraedom I hope so!

@arevellfraedom As soon as this build finishes, you should be able to grab the nuget from the artifacts. Let me know if you have any trouble accessing it, and I'll post it on the PR instead. https://dev.azure.com/xamarin/public/_build/results?buildId=14812&view=results